WordPress/Classes/ViewRelated/Me/My Domains/MyDomainsViewController.swift

```diff
@@ -0,0 +1,191 @@
+import UIKit
+import Combine
+
+final class MyDomainsViewController: UIViewController {
+    enum Action: String {
+        case addToSite
+        case purchase
+        case transfer
+
+        var title: String {
+            switch self {
+            case .addToSite:
+                return NSLocalizedString(
+                    "domain.management.fab.add.domain.title",
+                    value: "Add domain to site",
+                    comment: "Domain Management FAB Add Domain title"
+                )
+            case .purchase:
+                return NSLocalizedString(
+                    "domain.management.fab.purchase.domain.title",
+                    value: "Purchase domain only",
+                    comment: "Domain Management FAB Purchase Domain title"
+                )
+            case .transfer:
+                return NSLocalizedString(
+                    "domain.management.fab.transfer.domain.title",
+                    value: "Transfer domain(s)",
+                    comment: "Domain Management FAB Transfer Domain title"
+                )
+            }
+        }
+    }
+
+    // MARK: - Dependencies
+
+    private let viewModel: ViewModel
+
+    // MARK: - Views
+
+    private let tableView = UITableView(frame: .zero, style: .insetGrouped)
+
+    // MARK: - Properties
+
+    private lazy var state: ViewModel.State = viewModel.state
+
+    // MARK: - Observation
+
+    private var cancellable = Set<AnyCancellable>()
+
+    // MARK: - Init
+
+    init(viewModel: ViewModel = .init()) {
+        self.viewModel = viewModel
+        super.init(nibName: nil, bundle: nil)
+    }
+
+    required init?(coder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+
+    // MARK: - View Lifecycle
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        title = Strings.title
+
+        WPStyleGuide.configureColors(view: view, tableView: nil)
+        navigationItem.rightBarButtonItem = .init(systemItem: .add)
+        let addToSiteAction = menuAction(withTitle: Action.addToSite.title) { action in
+            print("add to site")
+        }
+        let purchaseAction = menuAction(withTitle: Action.purchase.title) { action in
+            print("purchase")
+        }
+        let transferAction = menuAction(withTitle: Action.transfer.title) { action in
+            print("transfer")
+        }
+
+        let actions = [addToSiteAction, purchaseAction, transferAction]
+        let menu = UIMenu(title: "", image: nil, identifier: nil, options: .displayInline, children: actions)
+        navigationItem.rightBarButtonItem?.menu = menu
```

Comment:
Same as this [comment](https://github.com/wordpress-mobile/WordPress-iOS/pull/21860/files#r1369490185). This code is related to the "+" menu context, and it will be removed.